src/tools/annotation/LengthTool.js

```diff
@@ -136,9 +137,27 @@ export default class LengthTool extends BaseAnnotationTool {
     // Calculate the length, and create the text variable with the millimeters or pixels suffix
     const length = Math.sqrt(dx * dx + dy * dy);
 
+    // Pixel diagonal
+    const uncertainty = colPixelSpacing
+      ? decimal(
+          colPixelSpacing * colPixelSpacing + rowPixelSpacing * rowPixelSpacing
+        ).sqrt()
+      : decimal.sqrt(2);
```

Comment:
This looks like code duplication. I suggest extracting the function to a utils file